lib/Net/DRI/DRD.pm

```diff
@@ -1528,11 +1528,11 @@ sub domain_check_claims
  if (defined $rd && exists $rd->{phase} && lc($rd->{phase}) ne 'claims')
  {
   # By default, most registries do NOT use a sub_phase is claims lookups. Therefore if you specify a phase it will be ignored
-  # Afilias/ARI/CentralNIC/CoreNic/CRR/Donuts/GMO/KS/PIR/RegBox/Rightside/StartingDot/Tango/UniRegistry
+  # Afilias/GoDaddy::DNRS/CentralNIC/CoreNic/CRR/Donuts/GMO/KS/PIR/RegBox/Rightside/StartingDot/Tango/UniRegistry
 
   # These registres use claims as phase + phase_name us sub_phase. domain_check_claims('test-validate.tube',{phase=>'landrush'});
-  # Neustar/MAM/FFM/KNet   (Knet seems to work either way - but rather put it here)
-  $lp->{sub_phase} = $rd->{phase} if ($bep =~ m/^(?:neustar|mam|ffm|knet)/);
+  # MAM/KNet   (Knet seems to work either way - but rather put it here)
```

Comment:
I think we sould remmove MAM - not in use anymore

Reply:
Removed all mam references as discussed